tests failed
```
======================================================================
FAIL: Suite should include remote-request if searching is configured
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_suite_remote_request.py", line 35, in test_remote_request
    self.assertXmlPartialEqual(self.get_xml('remote_request'), suite, "./remote-request[1]")
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/util.py", line 26, in assertXmlPartialEqual
    self.assertXmlEqual(expected, actual, normalize=False)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/util.py", line 32, in assertXmlEqual
    _check_shared(expected, actual, LXMLOutputChecker(), "xml")
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/util.py", line 126, in _check_shared
    raise AssertionError(message)
AssertionError: XML mismatch
--- want.xml
+++ got.xml
@@ -15,7 +15,6 @@
     <session>
       <query storage-instance="results" url="https://www.example.com/a/test_domain/phone/search/">
         <data key="case_type" ref="'case'"/>
-        <data key="include_closed" ref="'False'"/>
         <prompt key="name">
           <display>
             <text>
@@ -31,7 +30,7 @@
           </display>
         </prompt>
       </query>
-      <datum detail-confirm="m0_case_long" detail-select="m0_case_short" id="case_id" nodeset="instance('results')/results/case[@case_type='case']" value="./@case_id"/>
+      <datum detail-confirm="m0_case_long" detail-select="m0_case_short" id="case_id" nodeset="instance('results')/results/case[@case_type='case'][@status='open']" value="./@case_id"/>
     </session>
     <stack>
       <create>
----------------------------------------------------------------------
```

--- a/corehq/apps/app_manager/tests/data/suite/remote_request.xml
+++ b/corehq/apps/app_manager/tests/data/suite/remote_request.xml
@@ -17,6 +17,7 @@
       <query url="https://www.example.com/a/test_domain/phone/search/"
              storage-instance="results">
         <data key="case_type" ref="'case'"/>
+        <data key="include_closed" ref="'False'"/>
         <prompt key="name">
           <display>
             <text>
@@ -33,7 +34,7 @@
         </prompt>
       </query>
       <datum id="case_id"
-             nodeset="instance('results')/results/case[@case_type='case'][@status='open']"
+             nodeset="instance('results')/results/case[@case_type='case']"
              value="./@case_id"
              detail-confirm="m0_case_long"
              detail-select="m0_case_short"/>

